docs/content/dagster-cloud/managing-deployments/setting-up-alerts.mdx

```diff
@@ -85,6 +85,8 @@ Currently, Slack and email notifications are supported.
        height={511}
        />
 
+     To disconnect your Slack workspace: the Dagster Cloud Slack app will need to be removed from your Slack workspace. A Slack workspace admin can do that by going to the Dagster Cloud app's `Configuration` settings and selecting the **Remove App** button. After doing this, refresh the Alerts page and the button will show the **Connect to Slack** option again.
```

Comment:
We can actually simplify this a bunch by linking to Slack's documentation. Good for us in case things change on their end.

```suggestion
     **To disconnect Dagster Cloud from Slack**, remove the Dagster Cloud app from your Slack workspace. Refer to [Slack's documentation](https://slack.com/help/articles/360003125231-Remove-apps-and-custom-integrations-from-your-workspace#remove-an-app) for more info and instructions. Once the app is removed, refresh the **Alerts** page in Dagster Cloud and the **Connect to Slack** option will be displayed.
```